server/sonar-server/src/main/java/org/sonar/server/measure/live/LiveMeasureComputerImpl.java

```diff
@@ -131,7 +135,26 @@ public LiveMeasureComputerImpl(DbClient dbClient, IssueMetricFormulaFactory form
     matrix.getChanged().forEach(m -> dbClient.liveMeasureDao().insertOrUpdate(dbSession, m, null));
     projectIndexer.commitAndIndex(dbSession, singleton(project), ProjectIndexer.Cause.MEASURE_CHANGE);
 
-    return Optional.of(new QGChangeEvent(project, branch, lastAnalysis.get(), config, () -> Optional.of(evaluatedQualityGate)));
+    Metric.Level previousStatus = loadPreviousStatus(project, matrix);
+    return Optional.of(
+      new QGChangeEvent(project, branch, lastAnalysis.get(), config, previousStatus, () -> Optional.of(evaluatedQualityGate)));
+  }
+
+  @CheckForNull
+  private static Metric.Level loadPreviousStatus(ComponentDto project, MeasureMatrix matrix) {
+    return matrix.getMeasure(project, CoreMetrics.ALERT_STATUS_KEY)
+      .map(liveMeasureDto -> liveMeasureDto.getTextValue())
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. ([squid:S1612](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1612))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fsl%2FSONAR-10247%2Fwebhooks_called_only_if_qg_status_changes&issues=AWEEZ0czEFfjxkQcQAXt&open=AWEEZ0czEFfjxkQcQAXt)

[ISSUE_KEY]: AWEEZ0czEFfjxkQcQAXt